spec/models/webhooks/utilities_spec.rb

```diff
@@ -35,7 +35,9 @@
     # load dependent models
     module Testing
       include Webhooks::Utilities
-      EVENTS = %w[model_event model_event2].freeze
+      # rubocop:disable Lint/OrAssignmentToConstant
+      EVENTS ||= %w[model_event model_event2].freeze
```

Comment:
Should be do this instead?
```suggestion
      EVENTS ||= %w[model_event model_event2].freeze unless defined? EVENTS
```